provisioning/service/samples/readme.md

```diff
@@ -0,0 +1,78 @@
+# Samples for the Azure IoT Device Provisioning Service SDK for Node.js
+
+This file can be found in http://github.com/azure/azure-iot-sdk-node/provisioning/service/samples
+
+This folder contains simple samples showing how to use the various features of the Microsoft Azure IoT Device Provisioning Service from an application running JavaScript code.
+
+## List of samples
+
+* Create a simple IndividualEnrollment object.
+   *  [create_individual_enrollment.js][create-individual-enrollment]
+* Create a simple EnrollmentGroup object.
+   *  [create_enrollment_group.js][create-enrollment-group]
+* Create a couple of IndividualEnrollments and delete them.
+   *  [create_delete.js][create-delete]
+* Create queries for IndividualEnrollments, EnrollmentGroups and DeviceRegistrationStates that belong to an EnrollmentGroup.
+   *  [query.js][query-link]
+* Perform BulkEnrollmentOperations to do various CRUD operations.
+   *  [run_bulk_operation.js][run-bulk-operation]
+
+
+## How to run the samples
+
+In order to run the device samples you will first need the following prerequisites:
+* Node.js v4 or above on your target device. (Check out [Nodejs.org](https://nodejs.org/) for more info)
+* [Setup Azure IoT Device Provisioning ][lnk-setup-iot-provisioning] Stop before executing the the Create and provision a simulated device section.  The samples below will perform these steps.
+
+<<<<<<<<<<<<<<<<<<link to how to obtain an endorsment key!!!!>>>>>>>>>>>>>>>>>>
+
+Get the following files from the current folder:
+* [package.json][package-json]
+* **__sample_file.js__** (where **__sample_file.js__** is one of the files listed above and available in this folder)
+
+Place the files in the folder of your choice on the target machine/device then go through the following steps:
+
+* From a shell or Node.js command prompt, navigate to the folder where you placed the sample files. For creating an IndividualEnrollment do:
+
+```
+$ npm install
+$ node create_individual_enrollment.js "<the connection string for the Device Provisioning instance enclosed in quotes>" "<an endorsement key in quotes>"
+```
+
+<<<<<<<<<<<<<<<<<<<<<<<<<<<<<Link to how to create a signing certificate>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
```

Comment:
Dhanashri is working on this documentation

Reply:
What does creating a signing cert have to do with the sample flow? Should it be nested under the "For creating an EnrollmentGroup do" section?

Reply:
Yay!

---
In reply to: [150296581](https://github.com/Azure/azure-iot-sdk-node/pull/142#discussion_r150296581) [](ancestors = 150296581)